src/main/scala/org/broadinstitute/hail/variant/VariantSampleMatrix.scala

```diff
@@ -61,7 +62,7 @@ object VariantSampleMatrix {
         in => JsonMethods.parse(Source.fromInputStream(in).mkString)
       )
     } catch {
-      case e: Throwable => fatal(s"corrupt VDS: invalid metadata file: ${ e.getMessage }.\n  " +
+      case e: Throwable => fatal(s"corrupt VDS: invalid metadata file: ${e.getMessage}.\n  " +
```

Comment:
this formatting thing is Editor > Code Style > Scala > Other > Insert whitespaces in simple one line blocks